gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1038,6 +1039,55 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// This callback is called when the user is done matching transactions.
+static void
+gnc_ofx_match_done (GtkDialog *dialog, gint response_id, gpointer user_data)
+{
+    ofx_info* info = (ofx_info*) user_data;
+    GSList *iter = NULL;
+    
+    if (info->statement == NULL)
+        return;
+    
+    if (response_id == GTK_RESPONSE_OK)
+    {
+        char* message = ngettext ("Your statement includes balance information\nDo you want to reconcile now?",
+                                  "Your statements include balance information\nDo you want to reconcile now?",
```

Comment:
Punctuation: `Your statement includes ending balances. \n\nDo you want to reconcile now?`.

@gjanssens likes to minimise use of `gnc-warning` preferences; personally I think this is a valid use for it.

Reply:
Agreed :)